playbooks/build_image.yaml

```diff
@@ -0,0 +1,105 @@
+---
```

Comment:
My recommendation here would be to split up the `gce_pd`, `gce`, and the 'save the image' pieces into their own tasks files under the `openshift_gce` role so that they become functions.

Then just include them like this:
```
include_role:
  name: openshift_gce
  tasks_from: create_instance.yml
# install and configure the nodes
include_role:
  name: openshift_gce
  tasks_from:  save_image.yml
```
Call gce create_instance.yml which would call pd.yml and gce.yml to create disk and instance.  Then a separate call to seal up the image.

Thoughts?

Reply:
Yeah, I'll eventually do that.  Can be refactored under this, but I agree.